godot-bindings/src/godot_exe.rs

```diff
@@ -191,35 +170,35 @@ fn dump_header_file(godot_bin: &Path, out_file: &Path) {
     println!("Generated {}/gdextension_interface.h.", cwd.display());
 }
 
-fn patch_c_header(inout_h_path: &Path) {
+pub(crate) fn patch_c_header(in_h_path: &Path, out_h_path: &Path) {
     // The C header path *must* be passed in by the invoking crate, as the path cannot be relative to this crate.
     // Otherwise, it can be something like `/home/runner/.cargo/git/checkouts/gdext-76630c89719e160c/efd3b94/godot-bindings`.
 
-    println!("Patch C header '{}'...", inout_h_path.display());
+    println!("Patch C header '{}'...", in_h_path.display());
 
-    let mut c = fs::read_to_string(inout_h_path)
-        .unwrap_or_else(|_| panic!("failed to read C header file {}", inout_h_path.display()));
+    let mut c = fs::read_to_string(in_h_path)
+        .unwrap_or_else(|_| panic!("failed to read C header file {}", in_h_path.display()));
 
     // Detect whether header is legacy (4.0) format. This should generally already be checked outside.
     assert!(
         c.contains("GDExtensionInterfaceGetProcAddress"),
         "C header file '{}' seems to be GDExtension version 4.0, which is no longer support by godot-rust.",
-        inout_h_path.display()
+        in_h_path.display()
     );
 
     // Patch for variant converters and type constructors.
     c = c.replace(
         "typedef void (*GDExtensionVariantFromTypeConstructorFunc)(GDExtensionVariantPtr, GDExtensionTypePtr);",
         "typedef void (*GDExtensionVariantFromTypeConstructorFunc)(GDExtensionUninitializedVariantPtr, GDExtensionTypePtr);"
     )
-    .replace(
-        "typedef void (*GDExtensionTypeFromVariantConstructorFunc)(GDExtensionTypePtr, GDExtensionVariantPtr);",
-        "typedef void (*GDExtensionTypeFromVariantConstructorFunc)(GDExtensionUninitializedTypePtr, GDExtensionVariantPtr);"
-    )
-    .replace(
-        "typedef void (*GDExtensionPtrConstructor)(GDExtensionTypePtr p_base, const GDExtensionConstTypePtr *p_args);",
-        "typedef void (*GDExtensionPtrConstructor)(GDExtensionUninitializedTypePtr p_base, const GDExtensionConstTypePtr *p_args);"
-    );
+        .replace(
+            "typedef void (*GDExtensionTypeFromVariantConstructorFunc)(GDExtensionTypePtr, GDExtensionVariantPtr);",
+            "typedef void (*GDExtensionTypeFromVariantConstructorFunc)(GDExtensionUninitializedTypePtr, GDExtensionVariantPtr);"
+        )
+        .replace(
+            "typedef void (*GDExtensionPtrConstructor)(GDExtensionTypePtr p_base, const GDExtensionConstTypePtr *p_args);",
+            "typedef void (*GDExtensionPtrConstructor)(GDExtensionUninitializedTypePtr p_base, const GDExtensionConstTypePtr *p_args);"
+        );
```

Comment:
Is this re-indentation here necessary? Seems like rustfmt passed before :thinking: